### PR TITLE
Scroll containers shouldn't intercept pointer events

### PR DESCRIPTION
**Pull request by fsih**
### Resolves
Currently, the tool stops working when clicking over where the scroll bars would be when zoomed all the way out

### Proposed Changes
Make the scroll bar containers not take pointer events, so the tool can be used when the scrollbars aren't visible

### Reason for Changes
Odd that tools stopped working near the edges

### CODE REVIEW

File: src/components/scrollable-canvas/scrollable-canvas.jsx [hunk 1]
```diff
@@ -12,26 +12,34 @@ const ScrollableCanvasComponent = props => (
...
             <div
                 className={styles.horizontalScrollbar}
                 style={{
-                    width: `${props.horizontalScrollLengthPercent}%`,
-                    left: `${props.horizontalScrollStartPercent}%`,
-                    visibility: `${props.hideCursor ||
-                        Math.abs(props.horizontalScrollLengthPercent - 100) < 1e-8 ? 'hidden' : 'visible'}`
+                    'width': `${props.horizontalScrollLengthPercent}%`,
+                    'left': `${props.horizontalScrollStartPercent}%`,
+                    'pointer-events': 'all',
```
Comment (chrisgarrity) [inline src/components/scrollable-canvas/scrollable-canvas.jsx:24]: The [MDN spec](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events) for `pointer-events` says that 'all' only applies to svg content, but this style is getting applied to a `div`. Should the value be `auto` instead?

File: src/components/scrollable-canvas/scrollable-canvas.jsx [hunk 2]
```diff
@@ -12,26 +12,34 @@ const ScrollableCanvasComponent = props => (
...
                 className={styles.verticalScrollbar}
                 style={{
-                    height: `${props.verticalScrollLengthPercent}%`,
-                    top: `${props.verticalScrollStartPercent}%`,
-                    visibility: `${props.hideCursor ||
-                        Math.abs(props.verticalScrollLengthPercent - 100) < 1e-8 ? 'hidden' : 'visible'}`
+                    'height': `${props.verticalScrollLengthPercent}%`,
+                    'top': `${props.verticalScrollStartPercent}%`,
+                    'pointer-events': 'all',
+                    'display': `${props.hideCursor ||
```
Comment (chrisgarrity) [inline src/components/scrollable-canvas/scrollable-canvas.jsx]: why the change from visibility to display? Just wondering.

**Review by chrisgarrity — COMMENTED** [review]
Looks fine other than the question about the 'all' value.

File: src/components/scrollable-canvas/scrollable-canvas.jsx (hunk 2 above)
Comment (fsih) [reply src/components/scrollable-canvas/scrollable-canvas.jsx]: Display none will not intercept pointer events

File: src/components/scrollable-canvas/scrollable-canvas.jsx (hunk 1 above)
Comment (fsih) [reply src/components/scrollable-canvas/scrollable-canvas.jsx:24]: Thanks!

**Review by chrisgarrity — APPROVED** [review]
LGTM
